.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
While the `test.md` file was correctly added to the `.gitignore` file, the `node_modules` directory was not. Kindly fix this as it is a project requirement.

You may want to research the following Git command: `git rm -r --cached`.

Reply:
When installing Webhint and/or Stylelint or any other linter make sure you copy the `.github/workflows` directories into the project's root directory. After that execute the npm commands (I recommend you copy and paste them directly into your terminal from the README) and then copy the `hintrc` and `stylelintrc.json` files into the root directory as well. I hope this was useful.